deploy/scripts/src/mobile/utils/functions-mobile-journey.ts

```diff
@@ -48,70 +29,69 @@ function isStatusCode302 (res: Response): boolean {
   })
 }
 
-function isPageContentCorrect (res: Response, pageContent: string): boolean {
+function validatePageRedirect (res: Response, pageUrl: string): boolean {
```

Comment:
Suggest leaving in assertions around checking page content to ensure that the actual responses being returned are correct. The URL isn't sufficient to tell that the system is returning valid responses under load

Reply:
I've readded the page content assertions